File: Java/Reverse_String.java
void main() {
    char[] s = { 'h', 'e', 'l', 'l', 'o' };
    System.out.println(s);
    int left = 0, right = s.length - 1;
    while (left < right) {
        char temp = s[left];
        s[left] = s[right];
        s[right] = temp;
        left++;
        right--;
    }

    System.out.println(s);
}

File: Python/Reverse_String.py
def reverse_string():
    s = list("hello")
    print("".join(s))
    
    left = 0
    right = len(s) - 1
    
    while left < right:
        temp = s[left]
        s[left] = s[right]
        s[right] = temp
        left += 1
        right -= 1
        
    print("".join(s))

if __name__ == "__main__":
    reverse_string()
